requirements/run.txt

```diff
@@ -1,3 +1,3 @@
 numpy
-spyder >=5
```

Comment:
This should be removed since we are testing directly with master

Reply:
yes, I commented it out (so can be added again when 5 is there)